### PR TITLE
Toolbar title and subtitle

### PR DESCRIPTION
**Pull request by pawel-schmidt**

### CODE REVIEW

File: rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/RxToolbar.java [hunk 1]
```diff
@@ -41,6 +42,70 @@
...
   }
 
+  /**
+   * An action which sets the title property of {@code view} with character sequences.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult @NonNull
+  public static Action1<? super CharSequence> title(@NonNull final Toolbar view) {
```
Comment (JakeWharton) [inline rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/RxToolbar.java]: Can you rebase and add `@GenericTypeNullable` to this method?

File: rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/RxToolbar.java [hunk 2]
```diff
@@ -41,6 +42,70 @@
...
+  }
+
+  /**
+   * An action which sets the subtitle property of {@code view} with character sequences.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult @NonNull
+  public static Action1<? super CharSequence> subtitle(@NonNull final Toolbar view) {
```
Comment (JakeWharton) [inline rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/RxToolbar.java]: Can you rebase and add `@GenericTypeNullable` to this method?

File: rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxToolbar.java [hunk 3]
```diff
@@ -43,6 +44,70 @@
...
   }
 
+  /**
+   * An action which sets the title property of {@code view} with character sequences.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult @NonNull
+  public static Action1<? super CharSequence> title(@NonNull final Toolbar view) {
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxToolbar.java]: Can you rebase and add `@GenericTypeNullable` to this method?

File: rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxToolbar.java [hunk 4]
```diff
@@ -43,6 +44,70 @@
...
+  }
+
+  /**
+   * An action which sets the subtitle property of {@code view} with character sequences.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult @NonNull
+  public static Action1<? super CharSequence> subtitle(@NonNull final Toolbar view) {
```
Comment (JakeWharton) [inline rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxToolbar.java]: Can you rebase and add `@GenericTypeNullable` to this method?

**Comment by pawel-schmidt** [comment]
Rebased. BTW pretty useful annotation :)

**Comment by JakeWharton** [comment]
Merged manually. Thanks!

**Pull request closed without merging**